documentation/release_notes.rst

```diff
@@ -310,7 +310,7 @@ Known Issues and Limitations
 .. [#fnote1] The sorting algorithms in oneDPL use Radix sort for arithmetic data types compared with
    ``std::less`` or ``std::greater``, otherwise Merge sort.
 .. _`the oneDPL Specification`: https://spec.oneapi.com/versions/latest/elements/oneDPL/source/index.html
-.. _`the oneDPL Library Guide`: https://oneapi-src.github.io/oneDPL/index.html
+.. _`the Library Guide`: https://oneapi-src.github.io/oneDPL/index.html
```

Comment:
Please change to oneDPL Guide and remove "the" in the links.

Reply:
Done